ui/admin-portal/src/app/components/chat/create-update-post/create-update-post.component.ts

```diff
@@ -126,7 +126,7 @@ export class CreateUpdatePostComponent implements OnInit {
     this.emojiPickerVisible = false;
     const index: number = this.quillEditorRef.selection.savedRange.index;
     this.quillEditorRef.insertText(index, `${event.emoji.native}`, 'user');
-    this.quillEditorRef.setSelection(index + 2, 0);
+    this.quillEditorRef.setSelection(this.quillEditorRef.getLength(), 0);
```

Comment:
@cazcam34 - This is the right idea but I am not sure that it is replacing like for like.

The original code is inserting at the end of the current selection (it could be in the middle of a post).

Your code seems to always be inserting at the end of the whole data. 

Have you tested this when the emoji is being added in the middle of a post?

Reply:
Good point - but it's working fine as it's inserting the emoji at the index (see insertText) but after the insert it's just making sure that the selector goes to the end of the string. Beforehand the selector was also being put at the index + 2, and so if the emoji was longer it and another emoji was added without moving the selector it was inserting it at the wrong spot. @camerojo